<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-IUNQ6R05" name="QA测试" tests="2" failures="1" timestamp="10 Aug 2019 03:37:54 GMT" time="1.122" errors="0">
  <testcase name="testNormal" time="0.888" classname="mp.emp.EmpPageTest"/>
  <testcase name="testNormal" time="0.112" classname="mp.emp.EmpPageTest">
    <failure type="java.lang.AssertionError" message="响应取出的数据:ghj与期望结果不一致:CC expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: 响应取出的数据:ghj与期望结果不一致:CC expected [true] but found [false]
	at util.CheckUtil.checkContent(CheckUtil.java:172)
	at util.CheckUtil.checkJson(CheckUtil.java:156)
	at util.CheckUtil.checkOne(CheckUtil.java:121)
	at util.CheckUtil.check(CheckUtil.java:105)
	at mp.emp.EmpPageTest.testNormal(EmpPageTest.java:55)
	at com.sun.proxy.$Proxy0.invoke(Unknown Source)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- testNormal -->
</testsuite> <!-- QA测试 -->
